src/commands/cmd_slowlog.c

```diff
@@ -4,20 +4,60 @@
 * This file is available under the Redis Labs Source Available License Agreement
 */
 
-#include "cmd_context.h"
+#include "RG.h"
+#include "../redismodule.h"
 #include "../slow_log/slow_log.h"
+#include "../graph/graphcontext.h"
 
-void Graph_Slowlog(void *args) {
-	CommandCtx *command_ctx = (CommandCtx *)args;
-	RedisModuleCtx *ctx = CommandCtx_GetRedisCtx(command_ctx);
-	GraphContext *gc = CommandCtx_GetGraphContext(command_ctx);
+// usage:
+// GRAPH.SLOWLOG G
+// GRAPH.SLOWLOG G RESET
+int Graph_Slowlog
+(
+	RedisModuleCtx *ctx,
+	RedisModuleString **argv,
+	int argc
+) {
+	//--------------------------------------------------------------------------
+	// validations
+	//--------------------------------------------------------------------------
 
-	CommandCtx_TrackCtx(command_ctx);
+	ASSERT(ctx  != NULL);
+	ASSERT(argv != NULL);
+	if(argc < 2 || argc > 3) {
+		RedisModule_WrongArity(ctx);
+		return REDISMODULE_OK;
+	}
+
+	// get a hold of the graph key
+	RedisModuleString *key = argv[1];
+	GraphContext *gc = GraphContext_Retrieve(ctx, key, false, false);
+	if(gc == NULL) {
+		// if GraphContext is null, key access failed and an error been emitted
+		return REDISMODULE_OK;
```

Comment:
why return ok if it is error?

Reply:
we always need to return REDISMODULE_OK even if we fail